### PR TITLE
update generate.sh in mapper-framework

### PR DESCRIPTION
**Pull request by wbc6080**
**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
Previously, we needed to run the `make generate` command and then enter the name of the mapper to generate the mapper project. Now we can directly run `make generate <name>` to generate it. This modification can facilitate subsequent E2E testing.
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: staging/src/github.com/kubeedge/mapper-framework/hack/make-rules/generate.sh [hunk 1]
```diff
@@ -11,10 +11,14 @@ MAPPER_DIR="$(cd "$(dirname "$ROOT_DIR")" && pwd -P)"
 
 function entry() {
   # copy template
-  read -p "Please input the mapper name (like 'Bluetooth', 'BLE'): " -r mapperName
-  if [[ -z "${mapperName}" ]]; then
-    echo "the mapper name is required"
-    exit 1
+  if [ $# -eq 0 ] ;then
```
Comment (ZhengXinwei-F) [inline staging/src/github.com/kubeedge/mapper-framework/hack/make-rules/generate.sh:14]: You may write it this way: 
```
  mapperName=$1
  if [[ -z $mapperName ]]; then
    read -p "Please input the mapper name (like 'Bluetooth', 'BLE'): " -r mapperName
    if [[ -z "${mapperName}" ]]; then
      echo "the mapper name is required"
      exit 1
    fi
  fi
```

You can also use this to remind the user to enter their name:
```
  mapperName=$1
  while [[ -z $mapperName  ]];do
    read -p "Please input the mapper name (like 'Bluetooth', 'BLE'): " -r mapperName
  done
  echo $mapperName
```

Comment (wbc6080) [reply staging/src/github.com/kubeedge/mapper-framework/hack/make-rules/generate.sh:14]: I tried running the code you gave, but an error occurred

Comment (wbc6080) [reply staging/src/github.com/kubeedge/mapper-framework/hack/make-rules/generate.sh:14]: generate sh failed

**Comment by wbc6080** [comment]
Can you help to review it? This is the preparation for mapper e2e testing @WillardHu

**Comment by WillardHu** [comment]
> Can you help to review it? This is the preparation for mapper e2e testing @WillardHu

LGTM. But I‘m not a approver, you can wait for the approver.

**Comment by wbc6080** [comment]
> LGTM. But I‘m not a approver, you can wait for the approver.

In #5137，You have been added as approver of mapper-framework. For modifications to mapper-framework, you can add the approve tag if you like :) @cl2017 @WillardHu

**Pull request merged**